Syntaxes/Makefile.plist

```diff
@@ -743,6 +743,12 @@
 							<key>include</key>
 							<string>#interpolation</string>
 						</dict>
+						<dict>
+							<key>match</key>
+							<string>,</string>
+							<key>name</key>
+							<string>punctuation.separator.comma.makefile</string>
+						</dict>
```

Comment:
It works. The only thing is that it has to be added into 3 places:
1. function-variable-parentheses
2. function-variable-braces
3. `ifeq|ifneq|ifdef|ifndef` block

To avoid copy-paste it's better to define "comma" block at the beginning of the "repository" block, something like this in JSON:
```json
<key>comma</key>
<dict>
	<key>match</key>
	<string>,</string>
	<key>name</key>
	<string>punctuation.separator.comma.makefile</string>
</dict>
```
then just insert it into 3 places:
```xml
<dict>
	<key>include</key>
	<string>#comma</string>
</dict>
```